content/docs/guide.md

```diff
@@ -191,9 +191,8 @@ jobs:
 ```
 
 Pretty clear I'd say: when a new upstream release happens, propose it to
-dist-git main branch: Fedora Rawhide. Packit enables you to decide whether
-you want to do a direct push or create a pull request. If you want direct
-pushes, you need to set a global config option `create_pr` to `false`:
+dist-git main branch: Fedora Rawhide. When running locally, you can set `create_pr`
+option to `false`
```

Comment:
```suggestion
dist-git main branch: Fedora Rawhide. When running locally, you can set `create_pr`
option to `false` to directly push to dist-git instead of creating a pull-request.
```